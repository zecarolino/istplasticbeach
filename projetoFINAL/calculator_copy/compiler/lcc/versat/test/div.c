int div(int a, int b) {
	int x = 0;
	while (a >= b) a -= b, x++;
	return x;
}

int main() {
	return div(38, 5);
}
