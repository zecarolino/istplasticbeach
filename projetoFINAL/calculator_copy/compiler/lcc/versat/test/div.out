VCD info: dumpfile xtop.vcd opened for output.
7
Catched Trap at time  825 clock cycles:
